README.md

```diff
@@ -89,6 +89,11 @@ The following parameters may be set:
         <td>25080</td>
         <td>This is the port that the Audit Web Service is configured to listen on.</td>
       </tr>
+	  <tr>
+		<td>CAF_LOG_LEVEL</td>
+		<td>INFO</td>
+		<td>This environment variable can be used to override the current default logging level in Audit Web Service. The current default level is INFO.</td>
```

Comment:
I would change this to something like 'The logging level for the Audit Web Service. The supported logging levels include warn, error, info, trace, debug or all.'.